apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -238,4 +251,12 @@ export class TriggerEvent {
       status: 'subscriber_id_missing',
     };
   }
+
+  private modifyAttachmentNames(command: TriggerEventCommand) {
+    command.payload.attachments = command.payload.attachments.map((attachment) => ({
+      ...attachment,
+      file: Buffer.from(attachment.file, 'base64'),
+      name: `${command.organizationId}/${command.environmentId}/${hat()}/${attachment.name}`,
```

Comment:
This looks great @chavda-bhavik but a small question. When the file is sent to the user, does the fila name is "picture.jpg" or is it contains the whole path in the actually attachment name? 

Maybe we can add a new key on the model named `storagePath` or something similar and leave the name to be the name of the original file so it will be used when sending to the user?

Reply:
I can do it @scopsy. Thanks for your review.